hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -775,16 +775,6 @@ public PendingKeysDeletion getPendingDeletionKeys(
                   .addAllBlockIDs(blockIDS).build();
               int keyBlockSerializedSize = keyBlocks.getProto().getSerializedSize();
               serializedSize += keyBlockSerializedSize;
```

Comment:
Why have the serialized size calaculation?

Reply:
The function still has the ratisByteLimit in the function signature. Please raise a PR to revert this change separately as discussed earliser.

Reply:
Raised the revert [PR](https://github.com/apache/ozone/pull/8957).